docs/extensions/capabilities/common-capabilities.md

```diff
@@ -40,7 +40,7 @@ This extension can register custom app menus that will be displayed on OS native
 
 Example:
 
-``` typescript
+```typescript
```

Comment:
just curious, having a space is not OK? Seems to work fine

Reply:
It is fine, I was just trying to unify them.